src/System.Memory/src/System/Buffers/ReadOnlyBuffer.cs

```diff
@@ -0,0 +1,377 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#pragma warning disable 1591
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace System.Buffers
+{
+    /// <summary>
+    /// Represents a buffer that can read a sequential series of bytes.
```

Comment:
Doesn't ReadOnlyBuffer let us read non-contiguous memory as well (not just sequential)? Also, it is a generic type, so we should remove `bytes`.

Reply:
None of these comments were updated.